package.json

```diff
@@ -16,6 +16,8 @@
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
     "react-test-renderer": "^18.2.0",
+    "redux": "^4.2.0",
+    "redux-thunk": "^2.4.1",
```

Comment:
- [ ] When I run your application I got three error of dependencies but don't exist in the `package.json` file like `axios` and react-circular please try to fix that 
![image](https://user-images.githubusercontent.com/61790587/178518620-01d1f8db-f41e-46b8-9249-9cf7a42c52bc.png)